ConvertNumberToWords/Program.cs

```diff
@@ -14,38 +14,365 @@ static void Main()
+                {
+                    Console.WriteLine(testNumber + " " + CheckIfMinus(testNumber) + " " + ChangeTensToText(testNumber, FROM_NUMBER, TO_NUMBER));
+                }
+                //hundreds
+                else if ((testNumber >= HUNDREDS_FROM && testNumber < TENS_FROM) || (testNumber > TENS_TO && testNumber <= HUNDREDS_TO))
+                {
+                    Console.WriteLine(testNumber + " " + CheckIfMinus(testNumber) + " " + ChangeHundredsToText(testNumber, FROM_NUMBER, TO_NUMBER, TEENS_FROM, TEENS_TO, HUNDREDS_FROM, HUNDREDS_TO));
+                }
+                //thousands
+                else if ((testNumber >= THOUSANDS_FROM && testNumber < HUNDREDS_FROM) || (testNumber > HUNDREDS_TO && testNumber <= THOUSANDS_TO))
+                {
+                    Console.WriteLine("TUKSTANTINIAI  ");
+                    Console.WriteLine(testNumber + " " + CheckIfMinus(testNumber) + " " + ChangeThousandsToText(testNumber, FROM_NUMBER, TO_NUMBER, TEENS_FROM, TEENS_TO, TENS_FROM, TENS_TO,HUNDREDS_FROM,HUNDREDS_TO));
+                }
+
+            }
+             
+            else
+                resultFormat = "Incorrect format";
+            
+            Console.WriteLine("Format result "+ numberInput + " " + resultFormat + " " + CheckIfGoodNumber(numberInput, FROM_NUMBER, TO_NUMBER));
+            return numberText;
+        }
+
+        static bool CheckIfGoodNumber(string numberInput, int FROM_NUMBER, int TO_NUMBER)
+        {
+            /*
+            bool goodNumber = false;
+            for (int i = FROM_NUMBER; i <= TO_NUMBER; i++)
+            {
+
+                if(Convert.ToString(i) == numberInput)
+                {
+                    goodNumber = true;
+                }
+
+            }
+            return goodNumber;
+            */
+            /*
+             if (numberInput.All(char.IsDigit))
+                 goodNumber = true;
+             else
+                 goodNumber = false;
+
+             return goodNumber;
+             */
+
+            int i = 0;
+            bool result = int.TryParse(numberInput, out i);
+            return result;
+        }
+
+        static bool CheckIfNumberInRange(int testNumber, int FROM_NUMBER, int TO_NUMBER)
+        {
+            bool numberInRange = false;
+            if (testNumber>= FROM_NUMBER && testNumber <= TO_NUMBER)
+            {
+                numberInRange = true;
+
+            }
+            return numberInRange;
+        }
+
+        static string CheckIfMinus(int testNumber)
+        {
+            string minus = "";
+            if (testNumber < 0)
+                minus = "Minus";
+
+            return minus;
+        }
+        static string ChangeOnesToText(int testNumber, int FROM_NUMBER, int TO_NUMBER)
+        {
+            string ones = "";
+            if (testNumber == 0)
+                ones = "nulis";
+
+            else if (testNumber != 0 && testNumber <= TO_NUMBER && testNumber>= FROM_NUMBER)
```

Comment:
testNumber != 0 nebereikia tikrinti sitoje vietoje.
Is to kad jis failino if (testNumber == 0) naturaliai seka kad jis tikrai nebus lygus 0.